build-tools/stylelint/__tests__/no-motion-outside-of-mixin.test.js

```diff
@@ -64,29 +64,29 @@ describe('No motion outside of mixin', () => {
         }
       `);
       expect(result.errored).toBe(true);
-      expect(result.results[0].warnings).toEqual([
-        {
-          column: 15,
-          line: 5,
-          rule: '@cloudscape-design/no-motion-outside-of-mixin',
-          severity: 'error',
-          text: `Property "${property}" should be directly under 'with-motion' helper (@cloudscape-design/no-motion-outside-of-mixin)`,
-        },
-      ]);
+      expect(result.results[0].warnings.length).toBe(1);
```

Comment:
What changed here? New fields in the output?

1. Can we add them to the assertion? I vote for this if they are deterministic
2. Can we wrap the former assertion into `expect.objectContaining` to keep the former "single expect" form?

Reply:
P.S. everything below looks good and makes sense, thanks for upgrading